test-specifications/updateAccount.md

```diff
@@ -0,0 +1,126 @@
+
+**Transaction properties:**
+
+https://docs.hedera.com/hedera/sdks-and-apis/sdks/accounts-and-hbar/update-an-account
+
+**Crypto update protobufs:**
+
+https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_update.proto
+
+**Response codes:**
+
+[https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto](https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto)
+
+## Initialisation:
+
+```jsx
+new AccountUpdateTransaction();
+
+//example
+
+const transaction = new AccountUpdateTransaction().setAccountId(accountId).setKey(publicKey);
+```
+
+## Properties
+
+### **AccountId:**
+
+- The ID of the account to update.
+
+| Test no | Name                                                                             | Input                 | Expected response                                             | Implemented (Y/N) |
+|---------|----------------------------------------------------------------------------------|-----------------------|---------------------------------------------------------------|-------------------|
+| 1       | Update an account with no updates                                                | accountId, privateKey | The account update succeeds                                   | N                 |
+| 2       | Update an account with no updates without signing with the account's private key | accountId             | The account update fails with response code INVALID_SIGNATURE | N                 |
+
+
+### **Key:**
+
+- The public key to update on the account.
+
+| Test no | Name                                                       | Input                                                   | Expected response                                             | Implemented (Y/N) |
+|---------|------------------------------------------------------------|---------------------------------------------------------|---------------------------------------------------------------|-------------------|
+| 1       | Update an account with a new public key                    | accountId, oldPrivateKey, newPrivateKey, newPublicKey   | The account update succeeds and the account has the new key   | N                 |
+| 2       | Update an account key without signing with new private key | accountId, oldPrivateKey, newPublicKey                  | The account update fails with response code INVALID_SIGNATURE | N                 |
+| 3       | Update an account key and sign with incorrect private key  | accountId, oldPrivateKey, wrongPrivateKey, newPublicKey | The account update fails with response code INVALID_SIGNATURE | N                 |
```

Comment:
I opened a PR to add test cases that call out both key types (ECDSA and ED25519)